src/clj/rems/service/cache.clj

```diff
@@ -0,0 +1,191 @@
+(ns rems.service.cache
+  (:require [clojure.core.cache.wrapped :as cache]
+            [clojure.set :as set]
+            [com.rpl.specter :refer [ALL transform]]
+            [medley.core :refer [distinct-by]]
+            [rems.application.model :as model]
+            [rems.auth.util :refer [throw-forbidden]]
+            [rems.common.util :refer [index-by keep-keys]]
+            [rems.config :refer [env]]
+            [rems.db.applications :as applications]
+            [rems.db.attachments :as attachments]
+            [rems.db.blacklist :as blacklist]
+            [rems.db.catalogue :as catalogue]
+            [rems.db.form :as form]
+            [rems.db.licenses :as licenses]
+            [rems.db.users :as users]
+            [rems.db.workflow :as workflow]
+            [rems.dependencies :as dependencies]))
+
+(defn get-users []
+  (users/get-users))
+
+(defn get-user [userid]
+  (users/get-user userid))
+
+(defn user-exists? [userid]
+  (users/user-exists? userid))
+
+(defn get-users-with-role [role]
+  (set/union
+   (users/get-users-with-role role)
+   (applications/get-users-with-role role)))
```

Comment:
This should consider workflow handlers. Currently `applications/get-users-with-role` does not do it but could.

Reply:
There isn't a test that requires it at the moment. This is only user to enrich super users into applications which could use the "roles" table directly.